tox.ini

```diff
@@ -14,7 +14,14 @@ commands =
 deps =
     flake8==3.7.7
 
+[testenv:black]
```

Comment:
Nit. Could you put this above `[testenv:flake8]` so the environments are in alphabetical order?

Reply:
Good! It was changed.